src/web/CarplayWeb.ts

```diff
@@ -66,6 +67,17 @@ export default class CarplayWeb {
     driver.on('message', (message: Message) => {
       if (message instanceof Plugged) {
         this.clearPairTimeout()
+
+        const phoneTypeConfg = this._config.phoneConfig[message.phoneType]
+        if (phoneTypeConfg?.frameInterval) {
```

Comment:
Should we check for `this._frameInterval` being null here in addition to `phoneTypeConfg?.frameInterval != null`?

Reply:
Technically there should never be more than 1 plugged event - but we don't control that - so I think it's good to check - or perhaps always clear and reset the interval